lib/tasks/auth_source_external.rake

```diff
@@ -0,0 +1,12 @@
+namespace :auth_source_external do
+  desc 'Creates an external authentication source named "External", if no external authentication source exists.'
+  task :create => :environment do
+    User.as_anonymous_admin do
+      if AuthSourceExternal.any?
+        puts "Failed to create external authentication source. External authentication source named '#{AuthSourceExternal.pluck(:name)}' with id '#{AuthSourceExternal.pluck(:id)}' is already present."
+      else
+        AuthSourceExternal.create!(name: 'External')
+        puts 'Successfully created external authentication source.'
```

Comment:
missing `end` here

Reply:
Please forgive my mistakes in this PR. I was not in the correct mind.

Reply:
no worries, that's what we have code review and tests for :)